### ISSUE TITLE
[Question] Fuse bits 

### ISSUE THREAD

**Issue by Gloglo-ctr** [Atarity/mixtape-alpha#2]
I couldnt find an answer to this on the openmusiclabs wiki on the mixtape-alpha page. Can the factory fuse bit settings of the ATmega328P-au be used (L 0x62 H 0xD9 E 0xFF), or should they be changed?

**Comment by Atarity** [comment]
Yes, low fuse must be changed. So resulting command is `-U lfuse:w:0xE2:m -U hfuse:w:0xD9:m -U efuse:w:0xFF:m`

I've got same troubles with fuses as you. It was not documented by some reason. Here it is [the short article about my experience with Mixtape Alpha](https://snnkv.com/projects/ng-parts-mixtape-alpha/). I believe you can just translate it "in browser" flawlessly.

**Comment by Gloglo-ctr** [comment]
Thank you so much for the fast answer! Will look at your article.

Edit: After reading it, I had the same thought as you. It‘s hinted in the wiki that the clock should run at 8MHz, but I didnt want to solder the Atmega just to try it out. Hope other people can find this thread when they‘re looking for this answer.
